host/traceh.c

```diff
@@ -331,18 +331,43 @@ oe_result_t oe_log_set_callback(void* context, oe_log_callback_t callback)
     return OE_UNEXPECTED;
 }
 
+#ifdef _WIN32
+int gettimeofday(struct timeval* tv, struct timezone* tzp)
+{
+    static const uint64_t EPOCH = ((uint64_t)116444736000000000ULL);
```

Comment:
How was this EPOCH value chosen? Is this expected to be the same on all Windows Machines?

Reply:
EPOCH = 10000000 * 60 * 60 * 24 * 365 * (1970 - 1601) + 89. Yes this value is universal.

Reply:
Thanks, _time() is reused now.